src/script/user/UserMapper.ts

```diff
@@ -125,9 +127,7 @@ export class UserMapper {
       userEntity.accent_id(accentId);
     }
 
-    if (availability) {
```

Comment:
If availability was `0` (which is a possible value), the user would not be updated